apps/sms/views/conversation/style/message.css

```diff
@@ -35,11 +40,15 @@
 }
 
 .message.incoming .bubble {
+  background: #39c3c2;
+}
+html[dir="ltr"] .message.incoming .bubble {
   float: left;
-
   border-radius: 0 1.5rem 1.5rem 1.5rem;
-
-  background: #39c3c2;
+}
+html[dir="rtl"] .message.incoming .bubble {
```

Comment:
I don't know how `:-moz-dir(rtl)` works but I know for sure that adding another component to a selector has a performance impact.

I was just curious, I'm not saying one is better than the other. I guess that in _some_ cases it will be obvious that one is better than the other, and in most cases we don't really care.